pkg/codegen/generator/generator.go

```diff
@@ -0,0 +1,64 @@
+package generator
+
+import (
+	"path"
+	"strings"
+
+	"github.com/rancher/norman/generator"
+	"github.com/rancher/norman/types"
+)
+
+var (
+	outputDir  = "./"
+	baseCattle = "clients/rancher/generated"
+	// baseK8s    = "norman"
+)
+
+// GenerateClient is used to generate clients using `schema` an object of type Schemas from the norman/types package
+// defined by specific api e.g. management.cattle.io
+func GenerateClient(schemas *types.Schemas, backendTypes map[string]bool) {
+	version := getVersion(schemas)
+	group := strings.Split(version.Group, ".")[0]
+
+	cattleOutputPackage := path.Join(baseCattle, group, version.Version)
+
+	if err := generator.GenerateClient(schemas, backendTypes, outputDir, cattleOutputPackage); err != nil {
+		panic(err)
+	}
+}
+
+// func GenerateNativeTypes(gv schema.GroupVersion, nsObjs []interface{}, objs []interface{}) {
+// 	version := gv.Version
+// 	group := gv.Group
+// 	groupPath := group
+
+// 	if groupPath == "" {
+// 		groupPath = "core"
+// 	}
+
+// 	k8sOutputPackage := path.Join(outputDir, baseK8s, groupPath, version)
+
+// 	if err := generator.GenerateControllerForTypes(&types.APIVersion{
+// 		Version: version,
+// 		Group:   group,
+// 		Path:    fmt.Sprintf("/k8s/%s-%s", groupPath, version),
+// 	}, k8sOutputPackage, nsObjs, objs); err != nil {
+// 		panic(err)
+// 	}
+// }
```

Comment:
This part seems from the dev stage, would we need this part in the future?

Reply:
ah you're right, this wasn't needed anymore at least for now ... Since we're going to include wrangler anyways